content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -88,8 +88,6 @@ different Kubernetes components.
 | `CustomResourceValidationExpressions` | `true` | Beta | 1.25 | |
 | `DelegateFSGroupToCSIDriver` | `false` | Alpha | 1.22 | 1.22 |
 | `DelegateFSGroupToCSIDriver` | `true` | Beta | 1.23 | |
-| `DevicePlugins` | `false` | Alpha | 1.8 | 1.9 |
-| `DevicePlugins` | `true` | Beta | 1.10 | |
```

Comment:
It seems we have a new document in `command-line-tools-reference/` tracking the removed feature gate. This is likely the reason why the bot is complaining about merge conflict

Reply:
Thanks, rebased.